sql/src/main/java/org/apache/druid/sql/http/SqlResource.java

```diff
@@ -107,50 +105,45 @@ public Response doPost(
       try {
         return Response
             .ok(
-                new StreamingOutput()
-                {
```

Comment:
No changes other than converting to lambda. I was doing some experiments with adding a top level property to the json instead of using the queryContext, but went with the queryContext way and just left the lambda modification.